multiversx_sdk_cli/utils.py

```diff
@@ -37,9 +37,13 @@ def to_json(self):
 
 
 class BasicEncoder(json.JSONEncoder):
-    def default(self, o: Any):
+    def default(self, o: Any) -> Any:
         if isinstance(o, ISerializable):
             return o.to_dictionary()
+        if isinstance(o, bytes):
+            return o.hex()
+        if isinstance(o, list):
```

Comment:
Is this necessary? (I did not do an extra check).

Reply:
from what I've tested it's not. deleted it.